src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -56,6 +57,7 @@ public data class YamlConfiguration(
     internal val ambiguousQuoteStyle: AmbiguousQuoteStyle = AmbiguousQuoteStyle.DoubleQuoted,
     internal val sequenceBlockIndent: Int = 0,
     internal val allowAnchorsAndAliases: Boolean = false,
+    internal val maxAliasCount: Int = 100,
```

Comment:
> Rather than introducing a maxAliasCount property, what if we introduced a sealed class and used that to replace allowAnchorsAndAliases?

Sure, I'm all-in.
Just thought I do it in a backward compatible way. :-)

> I'd also prefer if maxAliasCount was a nullable unsigned integer - null indicating that there is no limit to the number of aliases allowed.

Definitely. Probably glimpsed too much at the other libs how they did it. :-)